client/asset/interface.go

```diff
@@ -568,6 +568,8 @@ type Balance struct {
 	// Locked is the total amount locked in the wallet which includes but
 	// is not limited to funds locked for swap but not actually swapped yet.
 	Locked uint64 `json:"locked"`
+	// Other is a place to list custom balance categories.
+	Other map[string]uint64 `json:"other"`
```

Comment:
I can see the need for this.  Has come up already.  What's the immediate first use that you have in mind out of curiosity? (none that I see in this PR)

Reply:
Oh, the fee balance for tokens?

Reply:
I was actually thinking about Zcash shielded balances and Decred mixing wallets.